improver_tests/psychrometric_calculations/cloud_top_temperature/test_CloudTopTemperature.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
```

Comment:
Two other cases to consider testing:

- Test an error is raised if the units of the temperature and ccl cubes are of a unit type that can't be converted (e.g. kg)
- Test an error is raised if ccl and temperature cubes have different spatial coordinates

Reply:
I don't know if I'm right, but...

- Adding these tests here would really be testing a method external to the plugin
- So I would be repeating tests that are covered elsewhere

However, I recognise that we do generally test for this in other plugins. This was one of the reasons I tried to standardise how we do these basic checks in plugins with an Abstract Base Class. I think it would be better to use a Mock test to assert that the relevant subfunction had been called, but I struggle to get Mock to do what I need.

Reply:
Hah! Since I wrote that, I've found out how to do it. Added as a separate test for clarity.